third-party-synthetic/active-gate-extensions/extension-third-party-ping/src/ping_extension.py

```diff
@@ -42,6 +44,8 @@ def query(self, **kwargs) -> None:
 
         target = self.config.get("test_target")
 
+        failure_count = self.config["failure_count"]
```

Comment:
use `self.config.get("failure_count", 1)` for  backward compatibility with possible old version upload to tenant and for consistency with other parameters

Reply:
I didn't use this code because I always set a default value in the json so it makes it redundant. But will change it though.